prime-dotnet-webapi/Migrations/20210920215613_PotentialPaperEnrolmentReturnee.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class PotentialPaperEnrolmentReturnee : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "LinkedErolmentId",
+                table: "Enrollee",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.InsertData(
+                table: "StatusReasonLookup",
+                columns: new[] { "Code", "Name" },
+                values: new object[,]
+                {
+                    { 19, "PRIME enrolment does not match paper enrollee record" },
+                    { 20, "Possible match with paper enrolment" }
```

Comment:
There are three new status reasons but only two in the migration. Could you please regenerate?

Reply:
Fixed